src/pages/lessons/fundamentals/eth-intro-part-3.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: A Developer's Guide to Ethereum, Pt. 3
+description: Introduction to Ethereum with web3.py and Python.
+icons: ["ethereum"]
+---
+
+import { LessonHeader } from "../../../components/mdx/LessonHeader";
+import Layout from "../../../components/Layout";
+
+<Layout
+    title="A Developer's Guide to Ethereum, Pt. 3"
+    description="Introduction to Ethereum with web3.py and Python."
+>
+<LessonHeader title="A Developer's Guide to Ethereum, Pt. 3" />
+
+Welcome to Part 3 in the saga! Part 1 covered blockchain fundamentals. Part 2 continued on to examine Ethereum accounts and how they enable participation in the network. Part 3 will build on those concepts with the introduction of smart contracts.
+
+## Smart contracts, an introduction
+
+The Ethereum blockchain has a great deal of value flowing through it. In previous articles, we discussed the flow of **ether** from one user to another via transactions. However, the network is capable of more sophisticated interactions, and those use cases are enabled by smart contracts.
```

Comment:
'The Ethereum blockchain has a great deal of value flowing through it, financial and otherwise' (might be a nice addidtion?)

Reply:
I would define this with 'the', so we're pointing to part 1 and 2 that you just referred to - `In 'the' previous lessons, ...`. (you chose 'lesson', as opposed to 'post' in part 2, if I remember correctly :))